src/lib/Client/EzRecommendationClientInterface.php

```diff
@@ -42,7 +42,10 @@ public function setUserIdentifier(string $userIdentifier): self;
 
     public function getUserIdentifier(): ?string;
 
-    public function sendRequest(string $method, UriInterface $uri, array $option = []): ResponseInterface;
+    /**
+     * @param array<string, array|string|bool> $options
```

Comment:
```suggestion
     * @param array<string, array|scalar> $options
```

Having `int|float` as options seems likely.